src/modules/UI/scenes/SendConfirmation/SendConfirmationConnector.js

```diff
@@ -5,8 +5,9 @@ import { connect } from 'react-redux'
 
 import { getCurrencyConverter, getExchangeRate } from '../../../Core/selectors.js'
 import type { Dispatch, State } from '../../../ReduxTypes'
+import { convertNativeToExchange } from '../../../utils'
 import { getExchangeDenomination, getSelectedCurrencyCode, getSelectedWallet } from '../../selectors.js'
-import { getDisplayDenomination } from '../../Settings/selectors.js'
+import * as SETTINGS_SELECTORS from '../../Settings/selectors.js'
```

Comment:
Use explicit imports. We are moving away from `import *` as it is harder to debug